src/defaultOperations.test.ts

```diff
@@ -0,0 +1,113 @@
+/*global jest*/
+/*eslint no-undef: "error"*/
+import DefaultOperations from "./defaultOperations";
+const axios = jest.requireActual("axios");
+jest.unmock("axios");
+
+const MockAdapter = require("axios-mock-adapter");
+const mockAxios = new MockAdapter(axios);
+
+const pageId = "12345";
+const apiKey = "##key##";
+const path = "my-path";
+const id = "23";
+
+test("I can import DefaultOperations", () => {
+  expect(DefaultOperations).toBeDefined();
+});
+
+test("I can require DefaultOperations", () => {
+  expect(require("./defaultOperations")).toBeDefined();
+});
+
+test("Default base url is set when built", () => {
+  expect(new DefaultOperations(null, null, null).BASE_URL).toBe(
+    "https://api.statuspage.io/v1/pages"
+  );
+});
+
+test("Default constructor parameters result attributes", () => {
+  const defaultOperations = new DefaultOperations("PageId", "ApiKey", "Path");
+  expect(defaultOperations.pageId).toBe("PageId");
+  expect(defaultOperations.path).toBe("Path");
+  expect(defaultOperations.config).toStrictEqual({
+    headers: {
+      Authorization: "OAuth ApiKey",
+    },
+  });
+});
+
+test("verify GET", async () => {
```

Comment:
I feel like the test's names doesn't really say anything.

At the very least it should be capitalized.